integration-libs/cdc/user-profile/update-email/cdc-update-email-component.service.ts

```diff
@@ -33,6 +40,9 @@ export class CDCUpdateEmailComponentService extends UpdateEmailComponentService
       authService,
       authRedirectService
     );
+    if (this.enableMyAccountV2) {
+      this.form.addControl('oldEmail', new UntypedFormControl(''));
+    }
```

Comment:
[MEDIUM]
Since we've already introduced property `updateSucceed$` and form control `oldEmail` to the parent class `UpdateEmailComponentService`, I believe we don't need to re-declare those things in the CDC-flavored child class? 
Perhaps I'm missing something obvious. My understand is that now all changes in `CDCUpdateEmailComponentService` could be reverted. Would you agree?

Note: analogically, my understanding is that all the changes in all the CDC-flavored component-services can be reverted, because the added logic there is already inherited from their relevant parent classes.

Reply:
Good points